### ISSUE TITLE
Display Grant Microphone Permission Text Only Once Per Session

### ISSUE THREAD

**Issue by Archna-1** [zorazrr/activ-card#34]
## Description
Right now, the grant permission text button is always displayed, even after the user clicks on it and has microphone permissions enabled.

## Todos

- [ ] Only display text if microphone permissions aren't enabled
- [ ] Prompt user to enable microphone permissions if they click on speak if not enabled (ideal behavior, stretch goal)
- [ ] Look into caching permissions (stretch goal and might not be possible on various browsers due to privacy)

## Useful links

- https://stackoverflow.com/questions/65488195/is-there-a-way-to-detect-the-browser-permissionsspecifically-camera-and-mic-in

## Screenshots

Before microphone access is enabled:
<img width="1728" alt="Screenshot 2024-02-24 at 3 02 15 AM" src="https://github.com/zorazrr/activ-card/assets/68453619/9079413b-7ded-4454-8c2c-b8f7fa542858">

After microphone access is enabled (notice text is still there):
<img width="1728" alt="Screenshot 2024-02-24 at 3 02 46 AM" src="https://github.com/zorazrr/activ-card/assets/68453619/25a23226-74c2-453e-887a-a36cc583c132">
